docs/content/docs/build/containers.md

```diff
@@ -3,28 +3,139 @@ title: Containers
 menuPosition: 2
 ---
 
+## Overview
+
+The container is the primary unit of encapsulation in Fluid. It consists of a collection of Loadable Objects and supporting lifecycle APIs. The container is represented as the `FluidContainer` type.
+
+The below docs are broken in to `Creation & loading` and `Interaction & lifecycle`.
+
 ## Creating & loading
 
-What is expected of the developer with respect to creating and deleting containers?
+Creation and loading happens from specific `[service]-client` packages. Each `[service]-client` package contains a common API structure around container interaction. The common API structure is around defining the container schema and getting the `FluidContainer` object.
+
+For the below scenarios we will be using the `client` to represent the `[service]-client` portion. For specific `[service]-client` usage see their corresponding documentation.
+
+### Container schema
+
+The container schema defines the data that the container instance knows about. It includes `initialObjects` that will synchronously loaded and always available. It also includes a definition of types that can be dynamically created. The same schema definition must be provided for both. For more information on `initialObjects` and dynamically cration see [Data modeling](./data-modeling.md)
+
+```typescript
+const schema = {
+    name: "example-container",
+    initialObjects: {
+        layout: SharedDirectory,
+        text: SharedString
+    },
+    dynamicObjectTypes: [ SharedCell, SharedString ],
+}
+```
```

Comment:
@tylerbutler, how do you read the word `or` in the above `creation *or* load` context. I read it as exclusive IE. it could be available when you create or when you load. But I think you mean it like. It is available when you create or when you load.

Reply:
No, I didn't mean it in the exclusive sense. Agree it's confusing. Maybe "which will be available immediately when the container is loaded?" In this case, the difference between loaded and created isn't relevant.